pennylane/fermi/fermionic.py

```diff
@@ -523,6 +630,36 @@ def to_mat(self, n_orbitals=None):
 
         return mat
 
+    def commute(self, fw, source, target):
+        r"""Apply anti-commuting relations until the operator in position source is in position target.
+
+        Args:
+            fw (FermiWord): The FermiWord within the FermiSentence that contains the operator to be commuted.
+
+            source (int): The position of the operator to be commuted.
+
+            target (int): The desired position of the operator occupying the source position.
+
+        Returns:
+            FermiSentence: The FermiSentence obtained after applying the anti-commutator relations.
+
+        **Example**
+
+        >>> w = FermiWord({(0, 0): '+', (1, 1): '-'})
+        >>> s = FermiSentence({w: 1})
```

Comment:
What's the `s = FermiSentence({w: 1})` for here?

Reply:
It should be `s.commute(w, 0, 1)` instead of `w.commute(0, 1)` on the next line.